packages/functions/src/main.ts

```diff
@@ -1,3 +1,12 @@
+import type { NetlifyGlobal } from '@netlify/types'
+
+declare global {
+  // Using `var` so that the declaration is hoisted in such a way that we can
+  // reference it before it's initialized.
```

Comment:
I don't think this is actually relevant here, so we can remove the comment and use `const`, right?